asmcomp/arm64/emit.mlp

```diff
@@ -1172,7 +1172,7 @@ let begin_assembly() =
      Alignment is needed to avoid linker warnings for
      shared_startup__code_{begin,end} (e.g. tests/lib-dynlink-pr4839).
    *)
-  if macosx then ` nop\n .align 3\n`;
+  if macosx then `	nop\n .align 3\n`;
```

Comment:
While here, the style prior to 5.0 looks to have been a single instruction per line, but regardless the `.align` should have tabs around it:
```suggestion
  if macosx then begin
    `	nop\n`;
    `	.align	3\n`
  end;
```
GitHub won't me comment on it at the correct line, but there is an existing `.align 3` above at L1154 which "should" also have a tab before the 3.

Reply:
Fixed both of these in the latest commit.